fmriprep/cli/parser.py

```diff
@@ -48,7 +48,8 @@ def _drop_sub(value):
 
     def _filter_pybids_none_any(dct):
         import bids
-        return {k: bids.layout.Query.ANY if v == "*" else v
+        return {k: bids.layout.Query.NONE if v is None else
+                (bids.layout.Query.ANY if v == "*" else v)
                 for k, v in dct.items()}
```

Comment:
some stylistic minor:
```suggestion
        return {
            k: bids.layout.Query.NONE
            if v is None
            else (bids.layout.Query.ANY if v == "*" else v)
            for k, v in dct.items()
        }
```

Reply:
+1 on this, greatly improves the readability